src/main/java/com/yelp/nrtsearch/server/luceneserver/field/IndexableFieldDef.java

```diff
@@ -344,4 +344,13 @@ public String getPostingsFormat() {
   public String getDocValuesFormat() {
     return docValuesFormat;
   }
+
+  /**
+   * Get the Lucene definition for this field.
+   *
+   * @return {link FieldType} for this field
+   */
+  public FieldType getFieldType() {
+    return fieldType;
+  }
```

Comment:
If we add other highlighters we might need to check for other options. E.g. unified highlighter supports both term vectors and index options. So I thought it better to just expose the fieldType rather than exposing multiple individual properties later.